brainz-mmd2-jaxb/src/main/java/org/musicbrainz/mmd2/Metadata.java

```diff
@@ -133,7 +132,6 @@
     "collectionList",
     "editorList",
     "entityList",
-    "editNote",
     "any"
```

Comment:
This pull request has likely not been generated on top of latest release [`v-2020-11-12`](https://github.com/metabrainz/mmd-schema/releases/tag/v-2020-11-12).